api_tests/lib/ledgers/ledger_runner.ts

```diff
@@ -25,15 +23,19 @@ export class LedgerRunner {
 
     constructor(
         private readonly projectRoot: string,
-        private readonly logDir: string
+        private readonly logDir: string,
+        private global: HarnessGlobal
```

Comment:
```suggestion
        private harness: HarnessGlobal
```
If we can't use the global `global` then let's not call it `global`.